### ISSUE TITLE
THE RESULTS IN THIS REPO ARE INVALID

### ISSUE THREAD

**Issue by dkobak** [wangxin0716/ssl#8]
This repo claims 92.9 accuracy and unfortunately is the first one in Google when one searches for `simclr cifar`.

However, there is a bug in linear evaluation (#1) which results in the backbone not been frozen during evaluation. This means that instead of training a linear readout, the entire network is getting fine-tuned. If one does the freezing properly, one only gets 88-89% with the training code from this repo.

Moreover, in the training code the repo does some weird implementation choices (#6) and uses suboptimal optimization parameters (#7).

A good implementation can get to 91.9 on CIFAR10 with ResNet18 (when training the linear readout with augmentations), see e.g. https://github.com/dkobak/simclr-cifar10.
